mk/buildroot.mk

```diff
@@ -25,10 +25,18 @@ BUILDROOT_SLIDES = \
 		buildroot-acknowledgements \
 		last-slides
 
-BUILDROOT_LABS = setup \
+BUILDROOT_BBB_LABS = setup \
 		buildroot-basic \
 		buildroot-rootfs \
 		buildroot-new-packages \
 		buildroot-advanced-packages \
 		buildroot-advanced \
 		buildroot-appdev
+
+BUILDROOT_STM32MP1_LABS = setup \
+		buildroot-basic-stm32 \
+		buildroot-rootfs-stm32 \
+		buildroot-new-packages \
+		buildroot-advanced-packages \
+		buildroot-advanced \
+		buildroot-appdev
```

Comment:
We have a few sections in each training that are board specific but that are named without a board suffix. EG we have `buildroot-basic` and `buildroot-basic-stm32`. It should be `buildroot-basic-bbb` for the first one. We'll have to fix this, but it can be done outside your series. Saying it out loud so we don't forget.